dist/constants.json

```diff
@@ -5883,931 +5883,931 @@
       },
       {
         "value": "436GF",
-        "label": "KALISPELL VA CLINIC"
+        "label": "Kalispell VA Clinic"
       },
       {
         "value": "436GH",
-        "label": "BENJAMIN C. STEELE VA CLINIC"
+        "label": "Benjamin Charles Steele VA Clinic"
       },
       {
         "value": "436GI",
-        "label": "GLASGOW VA CLINIC"
+        "label": "Glasgow VA Clinic"
       },
       {
         "value": "436GJ",
         "label": "MILES CITY VA CLINIC"
       },
       {
         "value": "436GK",
-        "label": "GLENDIVE VA CLINIC"
+        "label": "Glendive VA Clinic"
       },
       {
         "value": "436A4",
-        "label": "MILES CITY VA MEDICAL CENTER"
+        "label": "Miles City VA Community Living Center"
       },
       {
         "value": "436GL",
-        "label": "CUT BANK VA CLINIC"
+        "label": "Cut Bank VA Clinic"
       },
       {
         "value": "436GM",
-        "label": "LEWISTON CBOC"
+        "label": "Lewistown VA Clinic"
       },
       {
         "value": "436HC",
-        "label": "MERRIL LUNDMAN VA OPC"
+        "label": "Merril Lundman Department of Veterans Affairs Outpatient Clinic"
       },
       {
         "value": "436QB",
-        "label": "PLENTYWOOD VA CLINIC"
+        "label": "Plentywood VA Clinic"
       },
       {
         "value": "436QC",
-        "label": "HELENA VA CLINIC"
+        "label": "Helena VA Clinic"
       },
       {
         "value": "436QA",
-        "label": "HAMILTON VA CLINIC"
+        "label": "Hamilton VA Clinic"
       },
       {
         "value": "436GN",
-        "label": "DR. J MEDICINE CROW VA CLINIC"
+        "label": "Dr. Joseph Medicine Crow VA Clinic"
       },
       {
         "value": "436",
-        "label": "FORT HARRISON MEDICAL CENTER"
+        "label": "Fort Harrison Regional Benefit Office"
       }
     ],
     "ND": [
       {
         "value": "437GA",
-        "label": "GRAFTON VA CLINIC"
+        "label": "Grafton VA Clinic"
       },
       {
         "value": "437GB",
-        "label": "BISMARCK"
+        "label": "Bismarck VA Clinic"
       },
       {
         "value": "437GD",
-        "label": "MINOT CBOC"
+        "label": "Minot VA Clinic"
       },
       {
         "value": "437GI",
-        "label": "GRAND FORKS CBOC"
+        "label": "Grand Forks VA Clinic"
       },
       {
         "value": "437GF",
-        "label": "WILLISTON CBOC"
+        "label": "Williston VA Clinic"
       },
       {
         "value": "437GJ",
-        "label": "DICKINSON VA CLINIC"
+        "label": "Dickinson VA Clinic"
       },
       {
         "value": "437GK",
-        "label": "JAMESTOWN VA CLINIC"
+        "label": "Jamestown VA Clinic"
       },
       {
         "value": "437GL",
-        "label": "DEVILS LAKE VA CBOC"
+        "label": "Devils Lake VA Clinic"
       },
       {
         "value": "437",
-        "label": "FARGO VA HCS"
+        "label": "Fargo Regional Benefit Office"
```

Comment:
Let's confirm this name change.

This references a Benefit Office, not a Medical Facility.

Reply:
The new label for this facility code is **Fargo VA Medical Center**, but this is going from a Health Care System to a single facility. So let's hold off until we figure out how to resolve that.